docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
     
     ![](https://i.imgur.com/zFduIV8.jpg)
 
-4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+4 The automatically created account points to the Ethereum Mainnet by default (dropdown menu in the top right corner).
     
     ![](https://i.imgur.com/gk3TWUd.jpg)
 
-5. You can use the same account for different networks. For Waku we need to connect to the Sepolia test network.
-6. You can switch to a test network by clicking on the drop down menu. Select Sepolia test network for Waku:
+   You can use the same account for different networks. To publish to the Waku Network, we need to connect to the Sepolia test network.
+5. Switch to the Sepolia test network by selecting it from the dropdown menu. Ensure "Show test networks" is enabled.
 
-    ![](https://i.imgur.com/kbuup6y.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/670778eb-8bf0-42a6-8dd7-1dedfabeeb37)
 
-7. Click on Show/hide test networks.
-8. Enable “Show Test Networks".
+   The same account can be used with different networks. Note that the ETH balance is different for each network (each uses a different token).
```

Comment:
> The same account can be used with different networks.

Duplicates line 26.

> the ETH balance is different for each network (each uses a different token)

I'm concerned the word "token" might be confusing... Maybe call it "native token" to avoid the "USDT on ETH vs USDT on Tron" type of confusion people might be familiar with?

Reply:
Done in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890